content/en/tracing/troubleshooting/_index.md

```diff
@@ -108,6 +108,10 @@ Including metric partitions or grouping variables in service names instead of ap
 
  For example, instead of the service `web-store`, you might decide to name different instances of a service `web-store-us-1`, `web-store-eu-1`, and `web-store-eu-2` to see performance metrics for these partitions side-by-side. Datadog recommends implementing the **region value** (`us-1`, `eu-1`, `eu-2`) as a second primary tag.
 
+### Missing error message and stack trace
+
+Some traces that exhibit an error status, where the Errors tab of a trace will show a message reading 'Missing error message and stack trace' rather than an exception. A span is only expected to contain this information when an unhandled exception is traced. If an exception is handled in a try/catch block, `error.msg`, `error.type`, and `error.stack` span tags are not populated. If desired, these tags can be populated with [Custom Instrumentation](https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/?tab=datadogapi) code. It's also possible for spans to be given an error status without encountering an unhandled exception based on HTTP status code (400-599), this is a case where the same message is expected.
```

Comment:
```suggestion
In some traces with an error status, the **Errors** tab shows `Missing error message and stack trace` rather than exception details. 

A span can show this message for two possible reasons:
- The span contains an unhandled exception.
- An HTTP response within the span returned an HTTP status code between 400 and 599.

When an exception is handled in a try/catch block, `error.msg`, `error.type`, and `error.stack` span tags are not populated. To populate the detailed error span tags, use [Custom Instrumentation][18] code.
```
At the bottom of this file, please add the following line:
[18]: /tracing/trace_collection/custom_instrumentation/?tab=datadogapi